collections_hierarchy/mixins.py

```diff
@@ -1,87 +1,103 @@
 class ComparableMixin(object):
     def __eq__(self, other):
-        pass
```

Comment:
We weren't sure if it was cheating but we redefined the elements as data, and the getattr() function, then compared them.  But I like that you guys went down into it, also.  It looks good